packages/bot-strategy-runner/src/index.ts

```diff
@@ -0,0 +1,213 @@
+import program from "commander";
+import nodeFetch from "node-fetch";
+import assert from "assert";
+import * as fs from "fs";
+import * as cliProgress from "cli-progress";
+import * as bluebird from "bluebird";
+
+import { Logger, delay } from "@uma/financial-templates-lib";
+
+import { strategyRunnerConfig, buildBotConfigs, buildGlobalWhitelist, mergeConfig } from "./ConfigBuilder";
+import { runBot } from "./BotEntryWrapper";
+
+// Defaults for bot execution. If the user does not define these then these settings will be used.
+const defaultPollingDelay = 120;
```

Comment:
Why not 5 mins?

Reply:
can change it to that to be more consistant with the hub! I set it to this as the default polling delay within the bots is 2 mins.